scipy/optimize/tests/test_linprog.py

```diff
@@ -1312,6 +1314,33 @@ def test_bug_10124(self):
                       method=self.method, options=self.options)
         _assert_success(res, desired_x=[10, -3], desired_fun=-22)
 
+    def test_bug_10466(self):
+        """
+        Test that autoscale fixes poorly-scaled problem
+        """
+        c = [-8., -0., -8., -0., -8., -0., -0., -0., -0., -0., -0., -0., -0.]
+        A_eq = [[1., 1., 0., 0., 0., 0.,  0., 0., 0., 0., 0., 0., 0.],
+                [0., 0., 1., 1., 0., 0.,  0., 0., 0., 0., 0., 0., 0.],
+                [0., 0., 0., 0., 1., 1.,  0., 0., 0., 0., 0., 0., 0.],
+                [1., 0., 1., 0., 1., 0., -1., 0., 0., 0., 0., 0., 0.],
+                [1., 0., 1., 0., 1., 0.,  0., 1., 0., 0., 0., 0., 0.],
+                [1., 0., 0., 0., 0., 0.,  0., 0., 1., 0., 0., 0., 0.],
+                [1., 0., 0., 0., 0., 0.,  0., 0., 0., 1., 0., 0., 0.],
+                [1., 0., 1., 0., 1., 0.,  0., 0., 0., 0., 1., 0., 0.],
+                [0., 0., 1., 0., 1., 0.,  0., 0., 0., 0., 0., 1., 0.],
+                [0., 0., 1., 0., 1., 0.,  0., 0., 0., 0., 0., 0., 1.]]
+
+        b_eq = [3.14572800e+08, 4.19430400e+08, 5.24288000e+08,
+                1.00663296e+09, 1.07374182e+09, 1.07374182e+09,
+                1.07374182e+09, 1.07374182e+09, 1.07374182e+09,
+                1.07374182e+09]
+
+        with suppress_warnings() as sup:
+            sup.filter(OptimizeWarning, "Solving system with option...")
+            res = linprog(c, A_ub, b_ub, A_eq, b_eq, bounds,
+                          method=self.method, options=self.options)
+        assert_allclose(res.fun, -8589934560)
```

Comment:
(that's why I don't `_assert_success`)

Reply:
Adding to #9671 isn't really needed. ``_assert_success`` accepts `atol` and `rtol` which are then passed to ``assert_allclose``.

Reply:
No, I mean that when `linprog` checks the final solution for feasibility, it only does absolute tolerance checks. It would be nice to add relative tolerance checks so that it doesn't report status 4 incorrectly.